README-QUIC.md

```diff
@@ -89,10 +90,9 @@ This connects to a QUIC server using the specified ALPN protocol name and opens
 a single bidirectional stream. Data can be passed via stdin/stdout as usual.
 This allows test usage of QUIC using simple TCP/TLS-like usage.
 
-[openssl-quic(7) manpage]: https://www.openssl.org/docs/manmaster/man7/openssl-quic.html
+[openssl-quic(7) manual page]: https://www.openssl.org/docs/manmaster/man7/openssl-quic.html
```

Comment:
We still slightly disagree on the question whether the "manual page" should be part of the link text or not. But that's a matter of taste, I guess.  🤷‍♂️

Reply:
I would generally follow the principle that the noun phrase (with reasonable adjectives, and including definite article) is the best candidate for the link text. If for example I wanted to link to a product guide and it was obvious which one, I might have "see <a>the product guide</a>". 

So I would normally include "the", but since we have both "The openssl-quic(7) manual page" and "the openssl-quic(7) manual page", maybe this is the best way to do it